modules/phase_field/src/base/PhaseFieldApp.C

```diff
@@ -303,14 +325,11 @@ PhaseFieldApp::registerObjects(Factory & factory)
 
   registerFunction(ImageFunction);
 
+  registerVectorPostprocessor(GrainCentersPostprocessor);
+  registerVectorPostprocessor(GrainForcesPostprocessor);
+
   registerMesh(EBSDMesh);
   registerMesh(ImageMesh);
-
-#undef registerDeprecatedObjectName
-#define registerDeprecatedObjectName(obj, name, time) factory.regLegacyReplaced<obj>(stringifyName(obj), name, time)
-#undef registerObject
-#define registerObject(name) factory.regLegacy<name>(stringifyName(name))
-
```

Comment:
Check the latest devel, and copy the lines back into your PR. Preferable squash that into the commit that removed them!